galley/README.md

```diff
@@ -1,5 +1,8 @@
 # Galley
 
-Galley provides configuration management services for Istio.
-
-It is currently under early development.
+Galley is the top-level config ingestion, processing and distribution component of
+Istio. It is responsible for insulating the rest of the Istio components from the
+details of obtaining user configuration from Kubernetes. It contains Kubernetes
```

Comment:
Can we say "underlying platform" instead of Kubernetes specifically?

Reply:
Updated. PTAL.